.github/workflows/deploy_tre.yml

```diff
@@ -84,6 +84,9 @@ jobs:
 
           make all
 
+          az extension add --name azure-firewall
+          make tre-start
```

Comment:
You shouldn't need to explicitly run tre-start straight after a deployment - but no harm in running as it'll just check/report the current status.

It might be useful to separate out another make command that reports overall TRE status eg. `make tre-status`

Reply:
So I was thinking if App Gateway (or in future VMs) is deallocated and the TF hasn't touched it it may be stopped at this point.